arcgis-ios-sdk-samples/Maps/Apply scheduled updates to preplanned map area/ApplyScheduledUpdatesToPreplannedMapAreaViewController.swift

```diff
@@ -98,15 +98,11 @@ class ApplyScheduledUpdatesToPreplannedMapAreaViewController: UIViewController {
             if updatesInfo.downloadAvailability == .available {
                 let downloadSize = updatesInfo.scheduledUpdatesDownloadSize
                 let downloadSizeString: String
```

Comment:
There's no need to declare this ahead of time now. Please move the declaration to the line where it is now being initialized.